internal/provider/resource_consul_cluster_test.go

```diff
@@ -12,9 +12,10 @@ import (
 
 var consulCluster = `
 resource "hcp_consul_cluster" "test" {
-	cluster_id = "test-consul-cluster"
-	hvn_id     = hcp_hvn.test.hvn_id
-	tier       = "standard"
+	cluster_id         = "test-consul-cluster"
+	hvn_id             = hcp_hvn.test.hvn_id
+	tier               = "standard"
+	min_consul_version = "v1.10.6"
```

Comment:
When I tried this config, I got the error: 
```
specified Consul version (v1.10.6) is unavailable; must be one of: [v1.11.3 (recommended), v1.10.8, v1.9.15, v1.8.19]
```

Reply:
Updated the test to not hard code the version and instead use the `hcp_consul_versions` data source.